ts/Extensions/Annotations/Popup.ts

```diff
@@ -686,10 +686,14 @@ H.Popup.prototype = {
             });
 
             if (isRoot) {
-                storage = storage.sort(function (a): number {
-                    return (a as any)[1].match(/format/g) ? -1 : 1;
+                H.stableSort(storage, function (a: any): number {
+                    return a[1].match(/format/g) ? -1 : 1;
                 });
 
+                if (H.isFirefox) {
```

Comment:
Same here, we're transitioning away from using the namespace, so access `isFirefox` from Utils instead.

Reply:
It appears to me that `isFirefox` does not come from Utils yet.🤔